settings.gradle

```diff
@@ -26,3 +26,7 @@ include 'TabHostExample'
 include 'TabHostSelfContainedTabBrowsing'
 include 'wizard'
 include 'EditTextChips'
+include 'WearBuildConfigMobile'
+project(':WearBuildConfigMobile').projectDir = new File('WearBuildConfig/mobile')
```

Comment:
Sorry I wasn't clear enough: read my comment above as something in line with the suggestion of make this module a self-contained project. It's not _wrong_, it follows what's already in place, but I personally think it's a flawed approach that we should address soon.